mne/time_frequency/tests/test_spectrum.py

```diff
@@ -125,11 +125,15 @@ def test_n_welch_windows(raw):
     )
 
 
-def _get_inst(inst, request, evoked):
+def _get_inst(inst, request, evoked=None, average_tfr=None):
```

Comment:
This seems a bit safer then
```suggestion
def _get_inst(inst, request, *, evoked=None, average_tfr=None):
```